ddp-workspace/projects/ddp-testboston/src/assets/i18n/en.json

```diff
@@ -169,6 +170,16 @@
                     "Paragraphs": [
                         "Participants in the study can decide to stop being in the study at any time. Individuals who participate in the entire study can expect to complete the study after 6 months."
                     ]
+                },
+                {
+                    "Question": "Where can I find more information about COVID 19?",
+                    "Paragraphs": [],
+                    "Bullets": [
+                        "<strong>The US Centers for Disease Control and Prevention (CDC):</strong> The CDC has assembled a <a class=\"Link Link_semi-bold\" href=\"https://www.cdc.gov/coronavirus/2019-ncov/index.html\" target=\"_blank\">Coronavirus homepage</a> with a number of resources to learn about COVID-19 symptoms, how to protect yourself, and the most up to date national policies.",
+                        "<strong>Easy-to-read information with pictures:</strong> <a class=\"Link Link_semi-bold\" href=\"https://www.healthwise.net/partnersmychart/Content/StdDocument.aspx?DOCHWID=custom.partners.covidfamily\" target=\"_blank\">Caring for Yourself or a Family Member with COVID-19</a>.",
```

Comment:
Can we take a look at allowing for line breaks within the link. Currently the "Caring for Yourself" link goes to the next line to avoid line breaks. (Don't think big deal)

Reply:
On it